services/hanke-service/src/test/resources/fi/hel/haitaton/hanke/geometria/toinen_polygoni.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
These polygon json files could have more descriptive names to explain themselves / express purpose of use. Feel free to ignore.

Reply:
I tried to think of better names, but they are just three different polygons. I thought about using the coordinates of famous places in Helsinki and names after those, but I'm not sure having names like `polygon_stadikka.json`, `polygon_bollis.json` and `polygon_norris.json` will help much here. They might just raise more questions.